ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/set_properties.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
+# Directories
 export TOP_DIR=../../..
 export COMMON_DIR=../../../common
 export UBUNTU_COMMON_DIR=../../common
 export TEST_DIR=../../../tests
 export DISTRIBUTION=$(. /etc/os-release;echo $ID$VERSION_ID)
+
+# Environments
+export HPC_ENV=/opt/azurehpc
+
+# Component Versions
+export COMPONENT_VERSIONS=$(jq -r . $TOP_DIR/requirements.json)
```

Comment:
Is jq already installed by the time this script is called?

Reply:
Yes, jq now comes as a part of the base image